doc/handbook-general/company.md

```diff
@@ -9,40 +9,43 @@ title: Company
+If you are logged in as primary contact of your company you can reset these codes and thereby reassign them."
 
-![Number 4](../images/Numbers/4.png) Here an account ID is generated, which serves as identification of the company for the registration of the company to the ft.Helipad. This can be copied to the clipboard for manual transfer to the cash register.
+![Number 5](../images/Numbers/5.png) Here an account ID is generated, which serves as identification of the company for the registration of the company to the ft.Helipad. This can be copied to the clipboard for manual transfer to the cash register.
 
-![Number 5](../images/Numbers/5.png) Here an AccessToken - as access key for the logon of the company to the fiskaltrust.Helipad is generated. This can be copied to the clipboard for manual transfer to the cash register.
+![Number 6](../images/Numbers/6.png) Here an AccessToken - as access key for the logon of the company to the fiskaltrust.Helipad is generated. This can be copied to the clipboard for manual transfer to the cash register.
 
-![Number 6](../images/Numbers/6.png) The button \[Reset\] can be used to regenerate the account ID and the AccessToken.
+![Number 7](../images/Numbers/7.png) The button `[Reset]` can be used to regenerate the account ID and the AccessToken.
 
 #### Company employees
 
 Add new employees to your company and manage their permissions.
-![Company- Employees](images/AccountProfile/Employee.png)
+![Company- Employees](images/AccountProfile/Employee-with-primary.png)
 Company - Employees
 
-![Number 1](../images/Numbers/1.png) By default, the primary contact of the company is the user who registered the company as an employee.
+![Number 1](../images/Numbers/1.png) By default, the primary contact of the company is the user who registered the company as an employee. The highlighted record is the one with the primary role.
 
-![Number 2](../images/Numbers/2.png) By clicking on the arrow next to the pictograms to display the authorizations, the sliders for changing the authorizations are displayed.
+![Number 2](../images/Numbers/2.png) By clicking on the arrow next to the First name, the sliders for changing the authorizations are displayed.
 
 The different authorizations are described in chapter "[Employees of the company authorization administrations](#Employees-of-the-company-authorization-administration)".
 
 ![Number 3](../images/Numbers/3.png) In this case, the primary contact of the company is displayed. This user always has full rights. These authorizations cannot be reset either. Therefore the sliders are grey and cannot be changed.
 
-If no employee has the administrative rights, you lose the possibility to access thee fiskaltrust portal. In this situation the ft.team has to be contacted.
+If no employee has administrative rights, you lose the ability to access the fiskaltrust.Portal. In this situation fiskaltrust.Support has to be contacted.
```

Comment:
```suggestion
If no employee has administrative rights, you lose the ability to access the fiskaltrust.Portal. In this situation fiskaltrust.CustomerSuccess has to be contacted.
```

Reply:
Thanks, @AxelKutschera, though I think we'll need to revisit the terminology once more, as _fiskaltrust.Support_ is currently only in one document mentioned and _fiskaltrust.CustomerSuccess_ in _none_. Tagging monsieur @fabry-training as well 😇

Reply:
I liked to use Support. Because that's really a task done by the subdivision Support of the CustomerSuccess Team, isn't it?

Reply:
thank you, IMHO that is a question of terminology, another PBI - we follow that in another task